src/PlotKit.jl

```diff
@@ -69,7 +74,7 @@ export Color, RGBColor, RGBAColor
 export Point, LineStyle, Box, Box2
 
 # just data funs
-export norm, interp, star, triangle, oblong, curve_to_bezier, bezier_point,
+export norm, star, triangle, oblong, curve_to_bezier, bezier_point,
```

Comment:
nit: you might want to order them alphabetically (as the lists are quite long, it's easier to find them). Likewise for the other (re-)exports.